### ISSUE TITLE
Average mark or letter for each UV

### ISSUE THREAD

**Issue by allarddenis** [anaelChardan/Perso-MarksCrawler#4]
Hi, 

It would be great to calculate, for each UV, the average mark or equivalent letter (A, B, F...) considering associated coeffs.

xoxo

Denis

**Comment by anaelChardan** [comment]
Yup, nice comment ! I'll do it :-), you can contribute if you want :D

**Comment by anaelChardan** [comment]
In progress
